scripts/exrrfs_ioda_bufr.sh

```diff
@@ -2,31 +2,37 @@
 declare -rx PS4='+ $(basename ${BASH_SOURCE[0]:-${FUNCNAME[0]:-"Unknown"}})[${LINENO}]${id}: '
 set -x
 cpreq=${cpreq:-cpreq}
+
 cd ${DATA}
 
 # link the prepbufr file
-ln -snf ${OBSINprepbufr}/${CDATE}.rap.t${cyc}z.prepbufr.tm00 prepbufr
+${cpreq} ${OBSPATH}/${CDATE}.rap.t${cyc}z.prepbufr.tm00 prepbufr
 ${cpreq} ${EXECrrfs}/bufr2ioda.x .
+execfile=${HOMErrfs}/sorc/RDASApp/build/bin/bufr2ioda.x
 
 # generate the namelist on the fly
 REFERENCE_TIME="${CDATE:0:4}-${CDATE:4:2}-${CDATE:6:2}T${CDATE:8:2}:00:00Z"
 yaml_list=(
 "prepbufr_aircraft.yaml" 
-"prepbufr_ascatw.yaml" 
-"prepbufr_gpsipw.yaml" 
-"prepbufr_mesonet.yaml" 
-"prepbufr_profiler.yaml" 
-"prepbufr_rassda.yaml" 
-"prepbufr_satwnd.yaml" 
-"prepbufr_adpsfc.yaml" 
 "prepbufr_adpupa.yaml"
 )
+#yaml_list=(
+#"prepbufr_aircraft.yaml" 
+#"prepbufr_ascatw.yaml" 
+#"prepbufr_gpsipw.yaml" 
+#"prepbufr_mesonet.yaml" 
+#"prepbufr_profiler.yaml" 
+#"prepbufr_rassda.yaml" 
+#"prepbufr_satwnd.yaml" 
+#"prepbufr_adpsfc.yaml" 
+#"prepbufr_adpupa.yaml"
+#)
```

Comment:
Apologies for a picky comment. I just wanted to say that I *think* you can just comment out the specific lines you don't want in the yaml_list array instead of commenting the whole thing and creating a separate list. I also opened the file up and noticed there is trailing white space at the end of each of those lines. In fact there is a lot of trailing white space in several other files too. What are the thoughts on trailing white space? It might seem trivial, but seems like not quite the best practice to me. If using vim, you can configure your .vimrc to highlight trailing white space.

Reply:
Thanks, We can address those issues in the next PR after we can test the changes.